Zeeguu Reader/Zeeguu Reader/ArticleView.swift

```diff
@@ -32,13 +32,14 @@ class ArticleView: UIView {
 	
 	var article: Article!
 	var titleLabel: UILabel?
-	var contentView: UITextView?
+	var contentView: ZGTextView?
+	var delegate: ArticleViewDelegate?
 	
-	convenience init(article: Article) {
+	convenience init(article: Article, delegate: ArticleViewDelegate?) {
 		self.init()
 		self.article = article;
 		self.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
In earlier iOS days, you had something called autoresizing masks (and frames (origin and size)) that described how a view must be drawn to the screen. Nowadays, that is done by constraints (Auto Layout), however, you can still use the old way and by default the old way is translated to the new way (Auto Layout). But if you use Auto Layout yourself, the translation will cause conflicts and mess up your layout.

https://developer.apple.com/library/prerelease/ios/documentation/UIKit/Reference/UIView_Class/index.html#//apple_ref/occ/instp/UIView/translatesAutoresizingMaskIntoConstraints